package.json

```diff
@@ -1900,7 +1900,7 @@
         "build-ipywidgets-compile": "tsc -p ./src/ipywidgets && rimraf ./out/tsconfig.tsbuildinfo && node ./src/ipywidgets/scripts/copyfiles.js",
         "build-ipywidgets-webpack": "cross-env NODE_OPTIONS=--max_old_space_size=9096 webpack --config ./src/ipywidgets/webpack.config.js",
         "checkDependencies": "gulp checkDependencies",
-        "postinstall": "npm run download-api && node ./build/ci/postInstall.js",
```

Comment:
This should be done either as a separate job (optional that can fail).
Else we're all unable to submit PRs even when VS Code Insiders hasn't been updated.

Reply:
@joyceerhl @IanMatthewHuff /cc